### ISSUE TITLE
Getting "trustable": false

### ISSUE THREAD

**Issue by phildespres** [obrassard/shc-extractor#2]
Nice work/beau travail!

I am getting a "trustable": false when extracting data. I guess this part requires communication with covid19.quebec.ca ?

**Comment by obrassard** [comment]
Thanks ! Indeed, you are not the first asking me this question 😄 The Quebec government does not seem to have published  (yet ?) the public keys that allows authenticity validation of the information contained in the QR code payload. So for now it is "normal" that the server returns `"trustable": false`. According to the Smart Health Cards standard the public keys are usually published under the url `{issuer}/.well-known/jwks.json` (with the issuer corresponding to the `iss` field in the payload), however I guess they didn't publish them to avoid that anybody starts validating QR codes.

So basically, either they haven't published the keys, or they published them at another address that we don't know.

**Comment by phildespres** [comment]
En effet, je n'avais pas vu le billet fermé. Désolé!

**Comment by obrassard** [comment]
Aucun problème ! 😁
